runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java

```diff
@@ -0,0 +1,22 @@
+package net.runelite.client.plugins.clanchat;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup(
+	name = "Clan Chat",
+	keyName = "clanchat",
+	description = "Configuration for the clan chat plugin"
+)
+public interface ClanChatConfig extends Config
+{
+	@ConfigItem(
+		keyName = "showEnterAndLeaveMessages",
+		name = "Show member enter and leave messages",
+		description = "Configures whether or not to display messages informing who has entered and left the clan chat"
+	)
+	default boolean showEnterAndLeaveMessages() {
```

Comment:
^ Brace.

You can see these style errors by yourself if you enable the [Checkstyle flag](https://github.com/runelite/runelite/wiki/Checkstyle-with-IntelliJ-IDEA) in your maven build.

Reply:
It's strange how the build is passing with the checkstyle checks. It did give me a few errors which I sorted but it seemed to miss a bunch out.

Reply:
Yeah, I don't think our checkstyle catches extraneous indents at the moment, though it does detect misplaced braces.